tests/5.1/interop/test_interop_target.c

```diff
@@ -0,0 +1,66 @@
+//===--------------------- test_interop_target.c ----------------------------===//
+//
+// OpenMP API Version 5.1 Nov 2020
+// 
+// Test for interop construct utilizing target as the interopability. Runs
+// a OpenMP target construct and treating it as an external operation to test
+// if the interop contrruct properly executes and returns the result. Test
```

Comment:
construct ?

Reply:
It's listed as a construct in the spec, should I be using different terminology?

Reply:
I believe it was referring to a spelling mistake.